tests/buildall/config.yaml

```diff
@@ -0,0 +1,8 @@
+bobMinimumVersion: "0.25"
+
+layers:
+    - basement-gnu-linux
+    - name: basement
+      scm: git
+      url: ${BASEMENT_URL:-https://github.com/BobBuildTool/basement.git}
```

Comment:
Do we expect the URL/branch to be overridden? Or is this just some precaution?

Reply:
I used this for testing against basement changes (m4) and thought it might be useful.

Reply:
Fair enough.